.script/utils/dataConnector.ts

```diff
@@ -0,0 +1,189 @@
+export interface DataConnectorModel {
```

Comment:
what are all these definitions for? are they part of the validation?

Reply:
These are part of data connector model and we would needed for validation, as we add more test cases.

Reply:
OK. If possible try to avoid adding classes and type definitions pre-emptively for future test cases, since it is unclear why they are part of the current PR.